contrib/completion/zsh/_docker

```diff
@@ -641,6 +641,7 @@ __docker_subcommand() {
             _arguments \
                 $opts_help \
                 "($help -c --change)*"{-c,--change=-}"[Apply Dockerfile instruction to the created image]:Dockerfile:_files" \
+                "($help -m --message)"{-m,--message=}"[Set commit message for imported image]:message: " \
```

Comment:
The message can be set as the next argument. (@see example in https://github.com/monitorjbl/docker/blob/936b2c6afe490b562df4ebea854b116c3b40a9f5/docs/reference/commandline/import.md)